README.md

```diff
@@ -0,0 +1,29 @@
+# AccessControl
+
+## Getting started for development
+
+```
+$ docker compose up --build -d
+```
+
+This will create a `lib` container for the gem files and a `db` container for
+MariaDB, used as a test database.  Check if the database container was started
+with:
+
+```
+$ docker compose ps
```

Comment:
Typo

`docker compose` -> `docker-compose`

Reply:
Looks like `docker-compose` also adds a Docker sub-command `compose`, so it works just fine.

Reply:
Which version? The command `docker compose` does not work for me.